runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -75,10 +75,17 @@
 	private static final Pattern BINDING_USED_PATTERN = Pattern.compile(
 		"You bind the temple's power into (mud|lava|steam|dust|smoke|mist) runes\\.");
 	private static final String BINDING_BREAK_TEXT = "Your Binding necklace has disintegrated.";
+	private static final Pattern RING_OF_FORGING_CHECK_PATTERN = Pattern.compile(
+		"You can smelt ([0-9+]+|one) more pieces of iron ore before a ring melts\\.");
+	private static final Pattern RING_OF_FORGING_USED_PATTERN = Pattern.compile(
```

Comment:
This is a static string with no variance, so use a `String` and use `String#contains()` to check for it within the chat message subscriber. (see `BINDING_BREAK_TEXT`'s usage for an example)

```java
private static final String RING_OF_FORGING_USED_TEXT = "You retrieve a bar of iron.";
```

Reply:
Assuming the test is correct, `equals` should be fine.

Reply:
The same change can be made to `RING_OF_FORGING_USED_PATTERN`, it should be a String instead.